src/Lykke.Service.PayBackoffice/Areas/LykkePay/Views/PaymentRequests/PaymentRequestsPage.cshtml

```diff
@@ -38,6 +45,15 @@
                         divResult: '#paymentrequestsList'
                     });
                 });
+            var $statusSelect = $('#selectedStatus');
+            $statusSelect.select2().on("change",
+                function(e) {
+                    requests.doRequest({
+                        url: '@Url.Action("PaymentRequestsList")',
+                        formId: '#frmPaymentRequests',
+                        divResult: '#paymentrequestsList'
+                    });
+                });
             $('.select2-container--default').width(200);
             requests.doRequest({ url: '@Url.Action("PaymentRequestsList")', formId: '#frmPaymentRequests', divResult: '#paymentrequestsList' });
```

Comment:
this default request and `select2().on("change"` can be in one function, e.g. `getPayments()`